web/templates/invite_user_modal.hbs

```diff
@@ -56,6 +56,15 @@
     <div>
         <label>{{t "Streams they should join" }}</label>
         <div id="streams_to_add">
+            {{#if show_select_default_streams_option}}
+            <div class="select_default_streams new-style">
```

Comment:
Have used `new-style` class for now as it seemed easiest way to make the checkbox styling consistent with others. Would focus on removing the class, once I get done with release related work.

Reply:
Sounds reasonable.